.env.example

```
@@ -4,9 +4,14 @@ APP_ENV=development/production
 NEXT_PUBLIC_APP_ENV=development/production
 
 # API
+# Dev: http://localhost:1337
+# Prod: https://admin.scoutinglommel.be
 NEXT_PUBLIC_APP_BACKEND_URL=
+# Set SKIP_CODEGEN=1 to skip GraphQL codegen during `pnpm install` (e.g. in CI without a backend)
+# SKIP_CODEGEN=1
 STRAPI_API_TOKEN=
 NEXT_PUBLIC_STRAPI_UPLOAD_FILE_TOKEN=
+REVALIDATE_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REVALIDATE_SECRET key should go before the STRAPI_API_TOKEN key

(UnorderedKey)